pkg/ui/src/redux/locations.ts

```diff
@@ -14,7 +14,7 @@ export function selectLocationsRequestStatus(state: LocationState) {
 }
 
 export function selectLocations(state: LocationState) {
-  if (!state.cachedData.locations.valid) {
+  if (!state.cachedData.locations.data) {
```

Comment:
I would expect that the natural consequence of this strategy is that all the references to the `select...Status` type selectors should be removed.  It would seem that we never actually care about the `valid` prop the way things are currently set up.

Reply:
Yeah. Will leave to part of a broader refactor around indicating loading states.

Reply:
:+1: